spec/services/github_pull_request_review_service_spec.rb

```diff
@@ -39,6 +128,23 @@ def build(*_args)
       let(:repository) { create(:repository, tracked: true) }
 
       context "and review doesn't exist" do
+        before do
+          allow(service).to receive(:team_review_request_recommendations)
+            .and_return ({ best: [user1, user2, user3], worst: [user4, user5, user6] })
+          allow(pull_request).to receive(:pull_request_review_requests)
+            .and_return([
+                          double(pull_request_id: 123, github_user_id: 1),
+                          double(pull_request_id: 123, github_user_id: 6),
+                          double(pull_request_id: 123, github_user_id: 7)
+                        ])
+          allow(user1).to receive(:[]).and_return(user1.gh_id)
+          allow(user2).to receive(:[]).and_return(user2.gh_id)
+          allow(user3).to receive(:[]).and_return(user3.gh_id)
+          allow(user4).to receive(:[]).and_return(user4.gh_id)
+          allow(user5).to receive(:[]).and_return(user5.gh_id)
+          allow(user6).to receive(:[]).and_return(user6.gh_id)
```

Comment:
si se definieran en una lista esto podría reducirse a `users.each { |user| allow(user).to receive(:[]).and_return(user.gh_id) }`